types/eslint/index.d.ts

```diff
@@ -1316,14 +1276,90 @@ export namespace Linter {
          */
         settings?: Record<string, unknown>;
     }
+
+    /** @deprecated  Use `Config` instead of `FlatConfig` */
+    type FlatConfig = Config;
+
+    interface Globals {
+        [name: string]: boolean | "off" | "readonly" | "readable" | "writable" | "writeable";
```

Comment:
Following https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69957#discussion_r1670982112, one last string union here to extract if you'd like. It's not re-used anywhere that I know of.

Reply:
Ah, thanks. This union (including `boolean`) is defined in ESLint as [`GlobalConf`](https://github.com/eslint/eslint/blob/v9.6.0/lib/shared/types.js#L10), so I chose that name.